tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -78,7 +112,7 @@ module.exports = {
      * @param {String} role
      */
     selectRoleForNewCollaborator: function (role) {
-      role = role.replace(' ', '')
+      role = _(role).chain().toLower().startCase().replace(/\s/g, '').value()
```

Comment:
can you explain what happens here? maybe add a comment

Reply:
I thought it was pretty readable. Any suggestions to make it more readable?

Reply:
I'd hate comment here. Would this work?
```suggestion
      const titleCase = (string) => _(role).chain().toLower().startCase().value()
      role = _.replace(titleCase(role), /\s/g, '')
```